src/components/hv-view/index.js

```diff
@@ -40,6 +40,53 @@ export default class HvView extends PureComponent<HvComponentProps> {
 
   props: HvComponentProps;
 
+  isHiddenElement = (attributes: Attribute[]): boolean => {
+    for (let j = 0; j < attributes.length; j += 1) {
+      if (attributes[j].name === 'hide' && attributes[j].value === 'true') {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  canRenderElement = (element: Element): boolean => {
+    // ignore non rendering and hidden elements
+    return (
+      !!this.props.options.componentRegistry &&
+      !!element.namespaceURI &&
+      !!element.localName &&
+      !!this.props.options.componentRegistry[element.namespaceURI] &&
+      !!this.props.options.componentRegistry[element.namespaceURI][
+        element.localName
+      ] &&
```

Comment:
Are we assuming that if the component is part of the registry, it is rendering some content? I'm not sure that's always true/verifiable without actually getting the component to render.
Is there a risk in assuming that every child of a scroll view that has the `sticky` property will be a header that we want to render? I think this could simplify the logic quite a bit here. I might be missing something though on why we need to check first for non rendering elements.

Reply:
> Are we assuming that if the component is part of the registry, it is rendering some content

Yes, that is my understanding too. I manually tested it by adding non rendering elements like `<behavior>` as a child to sticky view to see if it's true. The only case where this is not true is when `hide="true"` hence I am specifically checking for hidden elements too.

> Is there a risk in assuming that every child of a scroll view that has the sticky property will be a header that we want to render?

I don't think so. If we add `sticky` to a non rendering element then it will be safely ignored.

> I might be missing something though on why we need to check first for non rendering elements

For example
```
  <view scroll="true">
    <view id="view-0">
      <text style="Text">Non Sticky</text>
    </view>
    <behavior />
    <view id="view-1" sticky="true">
      <text style="Text">Sticky</text>
    </view>
    <view id="view-2">
      <text style="Text">Non Sticky</text>
    </view>
  </view>
```
In this case when we want to pass `stickyHeaderIndices=[1]` as the prop to react, however if we don't filter out non rendering elements we would be passing `stickyHeaderIndices=[2]` because `<behavior />` will be counted too but when we render we ignore `<behavior />`. Thus instead of `view-1` being sticky `view-2` will become sticky which is not our intention.

Reply:
I see - I wonder if instead of iterating over the elements in the Hyperview DOM, we should iterate over those in the React DOM, that is the list of components returned by the call to `Render.renderChildren` method in `render()`. This would give us a more accurate reflection of the indexes we need to mark as sticky.
Each React component returned by `Render.renderChildren` should in theory have a reference it's Hyperview element. Let me know what you think.

Reply:
That's pretty neat, updated.